notebooks/stress-tests.ipynb

```diff
@@ -24,7 +24,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #174.        :param url: endpoint to stress test
</pre><p> </p><p>missing stress_test_progression in param doc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>